src/main/java/uk/nhs/hee/tis/usermanagement/service/CognitoAuthenticationAdminService.java

```
@@ -146,4 +153,20 @@ void deleteUser(AuthenticationUserDto authenticationUser) {
 
     cognitoClient.adminDeleteUser(request);
   }
+
+  @Override
+  public List<UserAuthEventDto> getUserAuthEvents(String username) {
+    List<UserAuthEventDto> events = new ArrayList<>();
+    try {
+      AdminListUserAuthEventsRequest request = new AdminListUserAuthEventsRequest()
+          .withUserPoolId(userPoolId)
+          .withUsername(username)
+          .withMaxResults(MAX_AUTH_EVENTS);
+      AdminListUserAuthEventsResult result = cognitoClient.adminListUserAuthEvents(request);
+      events = resultMapper.toUserAuthEventDtos(result.getAuthEvents());
+    } catch (AWSCognitoIdentityProviderException e) {
+      log.error(e.getMessage(), e);
+    }
+    return events;
```

I don't understand the rationale here.
I looked in the tests for what we might want to happen but don't see why we would want to proceed normally, ignoring errors if we hit an exception in either the request or extracting the result.

---

This is on me for not questioning things, I was following the approach of the other methods implemented in this service, but it would be good to be able to inform the user